test/integration/serverless/run.sh

```diff
@@ -167,7 +167,7 @@ functions_to_skip=(
 echo "Invoking functions for the first time..."
 set +e # Don't exit this script if an invocation fails or there's a diff
 for function_name in "${all_functions[@]}"; do
-    serverless invoke --stage "${stage}" -f "${function_name}" >/dev/null &
+    serverless invoke --stage "${stage}" -f "${function_name}" &>/dev/null &
```

Comment:
is this required? maybe it's better to not silent all errors?

Reply:
I added this because the Serverless Framework v3 seems to log some stuff to STDERR even when it successfully invokes functions, creating a bunch of noise. I'll look into this more and see if there is some way I can resolve it without silencing legitimate errors.

Reply:
I investigated this more -- I think we do want to hide the output here, because errors are normal when invoking these test functions. Some of the test functions are designed to intentionally throw errors or time out. If we don't hide the output here, it's quite confusing because it looks like something is going badly wrong with the tests when in fact everything is fine. With Serverless v3, we do need to suppress STDERR in addition to STDOUT to achieve this effect.